### DISCUSSION TITLE
不懂就问，自定义基建怎么实现训练室换干员

### DISCUSSION THREAD

**Discussion by KuiPer-LTW** [MaaAssistantArknights/MaaAssistantArknights#9324] [Q&A] [answered]
json怎么写？一图流上没有训练室选项
而MAA有此选项，还是说目前只有一个检查专精的功能？

**Comment by ABA2396** [comment, marked as answer]
训练室现在只有检测专精和继续专精，行为和 json 无关

**Reply by KuiPer-LTW** [reply]
哦 no，现在基建越来越多拖家带口的干员了，宿舍实在没空位，更何况我希望放到训练室的黍，大哥对她的判定甚至是不能在宿舍，sad

**Reply by KuiPer-LTW** [reply]
就人间烟火那沟槽的百分之20多效率，实在是放不进制造站，但是她又能给乌有提供5%订单效率
